truss/templates/control/control/application.py

```diff
@@ -22,7 +29,11 @@ def create_app(base_config: dict):
         app.config["inference_server_port"],
         app_logger=app.logger,
     )
-    patch_applier = PatchApplier(Path(app.config["inference_server_home"]), app.logger)
```

Comment:
For future cleanup, it would be nice to objectify this config instead of using dict keys.

Reply:
This is the standard flask config. I would love to look into objectifying it, could you point me to an example? Thanks

Reply:
I think `flask.config.Config.from_object` is what does this.  But really, we control the `create_app` method signature so we shouldn't need to be beholden to Flask contracts. 

From the Flask docs: 

```
An interesting pattern is also to use classes and inheritance for configuration:

class Config(object):
    DEBUG = False
    TESTING = False
    DATABASE_URI = 'sqlite://:memory:'

class ProductionConfig(Config):
    DATABASE_URI = 'mysql://user@localhost/foo'

class DevelopmentConfig(Config):
    DEBUG = True

class TestingConfig(Config):
    TESTING = True

To enable such a config you just have to call into [from_object()](https://flask.palletsprojects.com/en/0.12.x/api/#flask.Config.from_object)
```